PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -6,6 +6,8 @@
 //  Copyright © 2019 RevenueCat. All rights reserved.
 //
 
+#if !os(watchOS)
```

Comment:
Ok, here there is a x-files 😅 I've disabled all these test cases for `watchOS`. I'm not sure what it's happening, but all the tests with POST requests fail. It seems that the stub matcher doesn't recognize the request when is sent. But the same code works fine for the rest of the platforms 🕺🏼

TBH, I'm not sure that the `OHHTTPStubs` supports correctly` watchOS`. As you can see, there is an [issue] (https://github.com/AliSoftware/OHHTTPStubs/issues/287) with the same behavior and there is a [WIP PR] (https://github.com/AliSoftware / OHHTTPStubs / pull / 286) to support `watchOS`. Maybe I'm missing something 🤔

I've asked to `OHHTTPStubs` 'maintainers the status of that task ...

Reply:
let's add a comment to explain that

Reply:
Perfect! Anyway, I'm going to investigate a little bit more in order to know what it's happening 🤔